src/shared/modules/commands/helpers/grass.test.js

```diff
@@ -18,60 +18,85 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { parseGrass } from './grass'
+ /* global jest */
+
+import { fetchRemoteGrass, parseGrass } from './grass'
+
+jest.mock('services/remote', () => {
+  return {
+    get: (url) => {
+      return new Promise((resolve, reject) => {
+        resolve(url)
+      })
+    }
+  }
+})
+
+describe('Grass remote fetch', () => {
+  test('should not fetch from url not in the whitelist', () => {
```

Comment:
This test won't work if the function doesn't reject. Try changing `urlNotInWhitelist = 'foo'`.

Since Jest 20 you can return promises and expect it to reject. There's one missing peace though, you cannot match on error message. The workaround is to match an object and looks like this:

```
return expect(fetchRemoteGrass(urlNotInWhitelist, whitelist)).rejects.toMatchObject({
  message: 'Hostname is not allowed according to server whitelist'
})
```

https://facebook.github.io/jest/docs/expect.html#resolves and https://github.com/facebook/jest/issues/3601